train_segmentation_model.py

```diff
@@ -234,5 +247,10 @@ def train(gcp_bucket, config_file, random_module_global_seed, numpy_random_globa
         type=int,
         default=None,
         help='The setting of tf.random.set_seed(global seed), where global seed is int or default None (no seed given).')
+    argparser.add_argument(
+        '--pre-trained-model-id',
```

Comment:
sry too picky, but i'd simplify slightly to pretrained instead of pre-trained or pre_trained

Reply:
done